scala/src/main/org/apache/spark/api/csharp/SerDe.scala

```diff
@@ -267,6 +267,10 @@ object SerDe {
     out.writeInt(value)
   }
 
+  def writeLong(out: DataOutputStream, value: Long): Unit = {
```

Comment:
Should we start using this method @ https://github.com/Microsoft/SparkCLR/pull/303/files#diff-d078e99a5fd52f304c58828ec704b3cfL201?